tests/models/test_gptq_marlin.py

```diff
@@ -63,10 +70,11 @@ def test_models(
     gptq_marlin_model = vllm_runner(model_name=model_name,
                                     revision=revision,
                                     dtype=dtype,
-                                    quantization="marlin",
+                                    quantization="gptq",
                                     max_model_len=MAX_MODEL_LEN,
                                     tensor_parallel_size=1,
-                                    disable_custom_all_reduce=True)
+                                    disable_custom_all_reduce=True,
```

Comment:
to make this cleaner, can we remove `disable_custom_all_reduce`?

Reply:
Will try

Reply:
works